docs/source/aws-backend-requirements.rst

```diff
@@ -0,0 +1,39 @@
+Requirements
+------------
+
+Amazon Web Services
+~~~~~~~~~~~~~~~~~~~
+
+An Amazon Web Services account with sufficient funds must be available.
+
+The AWS credentials for the account must be present either in the environment as environment variables or in the default file system location under ``~/.aws/credentials``.
```

Comment:
For ``~/.aws/credentials``, use the [`:file:` directive](http://www.sphinx-doc.org/en/stable/markup/inline.html#role-file).